cpp_package/examples/get_data/src/multiple_streamers.cpp

```diff
@@ -0,0 +1,61 @@
+#include <iostream>
+#include <stdlib.h>
+#include <string>
+
+#ifdef _WIN32
+#include <windows.h>
+#else
+#include <unistd.h>
+#endif
+
+#include "board_shim.h"
+#include "data_filter.h"
+#include "ml_model.h"
+
+
+int main (int argc, char *argv[])
+{
+    BoardShim::enable_dev_board_logger ();
+    std::cout << "BoardShim Version: " << BoardShim::get_version () << std::endl;
+    std::cout << "DataFilter Version: " << DataFilter::get_version () << std::endl;
+    std::cout << "MLModel Version: " << MLModel::get_version () << std::endl;
+
+    struct BrainFlowInputParams params;
+    int board_id = (int)BoardIds::SYNTHETIC_BOARD;
+    int res = 0;
+
+    BoardShim *board = new BoardShim (board_id, params);
+
+
+    try
+    {
+        board->prepare_session ();
+        board->add_streamer ("file://streamer_default.csv:w");
+        board->add_streamer ("file://streamer_default.csv:w");
```

Comment:
actually this test should fail, you should stream to different files. I even not sure that its allowed to open the same file for writing in two places like this